validation/validation_efield_STEPS4/rallpack1.py

```diff
@@ -0,0 +1,324 @@
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+#
+# # Tests adapted from rallpack 1
+# # Original Rallpack 1 author: Iain Hepburn
+# # Test suite author: Alessandro Cattabiani
+#
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+from __future__ import print_function, absolute_import
+import steps.interface
+
+from steps.model import *
+from steps.geom import *
+from steps.rng import *
+from steps.sim import *
+from steps.saving import *
+
+import math
+import os
+import pandas as pd
+import sys
+import copy
+
+postproc_path = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))),
+    "postproc",
+)
+sys.path.append(postproc_path)
+from postproc.traceDB import TraceDB, Trace
+from postproc.comparator import Comparator
+
+from .. import configuration
```

Comment:
```suggestion
sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
from config import configuration
```
For some reason, keeping `from .. import configuration` yields a `ValueError: attempted relative import beyond top-level package`. It's probably linked to how unittest imports the module.

Reply:
usually importing from a folder that is root of the main script generates this problem